src/pymmcore_widgets/useq_widgets/_grid.py

```diff
@@ -368,3 +269,166 @@ def _on_change(self) -> None:
+        self.right.setValue(0)
+        self.right.setDecimals(3)
+        self.right.setButtonSymbols(QDoubleSpinBox.ButtonSymbols.NoButtons)
+        self.bottom = QDoubleSpinBox()
+        self.bottom.setRange(-10000, 10000)
+        self.bottom.setValue(0)
+        self.bottom.setDecimals(3)
+        self.bottom.setButtonSymbols(QDoubleSpinBox.ButtonSymbols.NoButtons)
+
+        self.lrtb_wdg = QWidget()
+        lrtb_grid = QGridLayout(self.lrtb_wdg)
+        lrtb_grid.setContentsMargins(0, 0, 0, 0)
+        lrtb_grid.addWidget(QLabel("Left:"), 0, 0, Qt.AlignmentFlag.AlignRight)
+        lrtb_grid.addWidget(self.left, 0, 1)
+        lrtb_grid.addWidget(QLabel("Top:"), 0, 2, Qt.AlignmentFlag.AlignRight)
+        lrtb_grid.addWidget(self.top, 0, 3)
+        lrtb_grid.addWidget(QLabel("Right:"), 1, 0, Qt.AlignmentFlag.AlignRight)
+        lrtb_grid.addWidget(self.right, 1, 1)
+        lrtb_grid.addWidget(QLabel("Bottom:"), 1, 2, Qt.AlignmentFlag.AlignRight)
+        lrtb_grid.addWidget(self.bottom, 1, 3)
+        lrtb_grid.setColumnStretch(1, 1)
+        lrtb_grid.setColumnStretch(3, 1)
+
+        self.bounds_layout = QFormLayout(self)
+        self.bounds_layout.addWidget(self.lrtb_wdg)
+
+        self.top.valueChanged.connect(self.valueChanged)
+        self.bottom.valueChanged.connect(self.valueChanged)
+        self.left.valueChanged.connect(self.valueChanged)
+        self.right.valueChanged.connect(self.valueChanged)
+
+
+class _ResizableStackedWidget(QStackedWidget):
+    def __init__(self, parent: QWidget | None = None) -> None:
+        super().__init__(parent=parent)
+        self.currentChanged.connect(self.onCurrentChanged)
+
+    def addWidget(self, wdg: QWidget) -> None:
+        wdg.setSizePolicy(QSizePolicy.Ignored, QSizePolicy.Ignored)
+        super().addWidget(wdg)
+
+    def onCurrentChanged(self, idx: int) -> None:
+        for i in range(self.count()):
+            wdg = self.widget(i)
+            if wdg is None:
+                continue
+            if i == idx:
+                wdg.setSizePolicy(QSizePolicy.Expanding, QSizePolicy.Expanding)
+            else:
+                wdg.setSizePolicy(QSizePolicy.Ignored, QSizePolicy.Ignored)
+            wdg.adjustSize()
+        self.adjustSize()
+
+
+class _BottomStuff(QWidget):
+    valueChanged = Signal()
+
+    def __init__(self) -> None:
+        super().__init__()
+
+        self.overlap_lbl = QLabel("Overlap:")
+        self.overlap = QDoubleSpinBox()
+        self.overlap.setRange(-1000, 1000)
+        self.overlap.setValue(0)
+        self.overlap.setSuffix(" %")
+        self.order_lbl = QLabel("Acquisition order:")
+        self.order = QEnumComboBox(self, OrderMode)
+        self.relative_to_lbl = QLabel("Current position:")
+        self.relative_to = QEnumComboBox(self, RelativeTo)
+
+        self.form_layout = QFormLayout(self)
+
+        # NB relative_to added in self.setMode
+        self.form_layout.addRow(self.overlap_lbl, self.overlap)
```

Comment:
if you want that short line here, which i also liked, add:

```suggestion
        self.form_layout.addRow("", SeparatorWidget())
```